src/redux/reducers/episodesReducer.js

```diff
@@ -0,0 +1,40 @@
+import {
+  FETCH_EPISODES_BEGIN,
+  FETCH_EPISODES_SUCCESS,
+  FETCH_EPISODES_FAILURE
+} from 'redux/actions/types';
+
+const initialState = {
+  episodes: [],
```

Comment:
Re-think this name of key. Now your state looks like this:
![screen shot 2018-10-05 at 07 55 03](https://user-images.githubusercontent.com/16289193/46518608-5eae2500-c875-11e8-8a58-fc6f8e21fe23.png)
And you can see there is episodes inside episodes. Maybe change into 'data' or 'all' ?